mcd/auctions.md

```diff
@@ -103,6 +106,17 @@ type Query {
      filter:    BidFilter
    ): [Flip]
 
+   activeFlips(
```

Comment:
Happy to add these queries, but wanted to point out that you should be able to filter on `dealt` == `false` - in case that might make this moot